monitoring/my-kube-prometheus/default.tmpl

```diff
@@ -0,0 +1,18 @@
+#See https://github.com/prometheus/alertmanager/issues/3310 for details
+
+{{ define "our.discord.title" }}
+    Unchained alert! {{ .GroupLabels.SortedPairs.Values | join " " }}
+{{ end }}
+
+{{ define "our.discord.message" }}
+    {{ range .Alerts.Firing }}
+      Alert: **{{ printf "%.150s" .Annotations.summary }}** ({{ .Labels.severity }})
+      Description: {{ printf "%.150s" .Annotations.description }}
+      Alertname: {{ .Labels.alertname }}
+      Namespace: {{ .Labels.namespace }}
```

Comment:
add service label? seeing them as empty on the alerts in the discord channel

Reply:
Cleaned this one up already but didn't apply it